gui/src/components/Backups.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+    <div>
+        <h4 class="py-3">Backups in Cluster: {{cluster.cluster_name}}</h4>
+        <ClusterSummary :cluster="cluster"/>
+        <div class="row justify-content-end mb-3 pr-3 py-3">
+            <button class="btn btn-primary mx-1" data-toggle="modal" data-target="#registerBackup">Create Backup
+            </button>
+            <button
+                    type="button"
+                    class="btn btn-primary mx-1"
+                    @click="viewRestoreHistory()"
+            >View Restores
+            </button>
+        </div>
+        <table class="table text-center table-bordered">
```

Comment:
I realized that all table entries might be better to be left aligned instead of centered.
![segment-view](https://user-images.githubusercontent.com/1052473/86303680-8b0ff480-bc47-11ea-9b38-5cdb27aace4c.png)